python_scripts/linear_models_regularization.py

```diff
@@ -287,7 +287,7 @@
 # ```
 #
 # In the previous analysis, we did not study if the parameter `alpha` will have
-# an effect on the performance. We chose the parameter beforehand and fix it
+# an effect on the performance. We choose the parameter beforehand and fix it
```

Comment:
```suggestion
# an effect on the performance. We chose the parameter beforehand and fixed it
```

Reply:
This line should be in past tense, as we are referring to "the previous analysis"